kong/tracing/propagation.lua

```diff
@@ -124,7 +125,7 @@ local function parse_zipkin_b3_headers(headers, b3_single_header)
 
   local trace_id_header = headers["x-b3-traceid"]
   if trace_id_header and ((#trace_id_header ~= 16 and #trace_id_header ~= 32)
-                           or trace_id_header:match("%X")) then
+    or trace_id_header:match("%X")) then
```

Comment:
I think this line should algin to `((#trace_id_header ...`, just like the original form.

Reply:
resolved